package/MDAnalysis/lib/transformations.py

```diff
@@ -1272,11 +1297,28 @@ def quaternion_matrix(quaternion):
     q = np.outer(q, q)
     return np.array(
         (
-            (1.0 - q[2, 2] - q[3, 3], q[1, 2] - q[3, 0], q[1, 3] + q[2, 0], 0.0),
-            (q[1, 2] + q[3, 0], 1.0 - q[1, 1] - q[3, 3], q[2, 3] - q[1, 0], 0.0),
-            (q[1, 3] - q[2, 0], q[2, 3] + q[1, 0], 1.0 - q[1, 1] - q[2, 2], 0.0),
-            (0.0, 0.0, 0.0, 1.0)
-        ), dtype=np.float64)
```

Comment:
What about this one? If find the old version much more difficult to read, but the formatting seems intentional. It makes it clear it is a 4x4 matrix, but reading the elements is super difficult since they are quite complex.

Reply:
Both types are kinda hard to read here imho :/

Probably the new one gets my vote too? It's easier to see the `,`